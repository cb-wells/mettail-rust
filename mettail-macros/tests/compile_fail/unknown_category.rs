use mettail_macros::theory;

theory! {
    name: Invalid,
    
    exports {
        Proc
    }
    
    terms {
        // Error: 'Name' is not exported
        Quote . Name ::= "@" (Proc) ;
    }
}

fn main() {}

