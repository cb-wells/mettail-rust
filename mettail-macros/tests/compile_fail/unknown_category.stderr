error: expected identifier
  --> tests/compile_fail/unknown_category.rs:12:30
   |
12 |         Quote . Name ::= "@" (Proc) ;
   |                              ^
